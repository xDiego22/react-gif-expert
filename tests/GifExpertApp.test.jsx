import { render, screen } from "@testing-library/react";
import { GifExpertApp } from "../src/GifExpertApp";

describe('prueba en componente <GifExpertApp/>', () => {

    test('debe hacer match con el snapshot', () => { 
        const { container } = render(<GifExpertApp />);
        expect(container).toMatchSnapshot();
    })
    
    test('debe mostra el titulo GifExpertApp', () => { 

        render(<GifExpertApp />);

        expect(screen.getByText('GifExpertApp')).toBeTruthy();

     })
});