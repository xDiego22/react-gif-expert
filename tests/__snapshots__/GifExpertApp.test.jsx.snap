// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`prueba en componente <GifExpertApp/> debe hacer match con el snapshot 1`] = `
<div>
  <h1>
    GifExpertApp
  </h1>
  <form
    aria-label="form"
  >
    <input
      placeholder="buscar gifs"
      type="text"
      value=""
    />
  </form>
  <h3>
    One Punch
  </h3>
  <h2>
    Cargando...
  </h2>
  <div
    class="card-grid"
  />
</div>
`;
